core/src/main/java/org/apache/calcite/rel/rules/FilterProjectTransposeRule.java

```diff
@@ -258,5 +261,14 @@ default Config withOperandFor(Class<? extends Filter> filterClass,
                   b2.operand(relClass).anyInputs())))
           .as(Config.class);
     }
+
+    /** Limit how much complexity can increase during merging.
+     * Default is {@link RelOptUtil#DEFAULT_BLOAT} (100). */
```

Comment:
Nit: Probably better to omit the value itself as one can easily look it up with the IDE. The risk I see is that, in case the value gets changed in the code, we end up with an inconsistency.

Reply:
Removed